dev/import-beats/fields.go

```diff
@@ -72,3 +73,17 @@ func loadDatasetFields(modulePath, moduleName, datasetName string) ([]byte, erro
 	}
 	return buffer.Bytes(), nil
 }
+
+func loadEcsFields(ecsDir string) ([]fieldsTableRecord, error) {
+	ecsFieldsPath := filepath.Join(ecsDir, "generated/beats/fields.ecs.yml")
```

Comment:
Good for now but at some point we should start to depend on a specific release of ECS and not master. Best same version Beats is on.

Reply:
At the moment, we can switch branches manually on the ECS side.